README.md

```diff
@@ -0,0 +1,161 @@
+    getcapsule/dpdk-devbind:18.11.6 \
+    /bin/bash -c 'dpdk-devbind.py --force -b uio_pci_generic #PCI_ADDR#'
+```
+
+Once you've made the necessary changes, you can pull down the sandbox container and run it,
+
+```
+host$ docker pull getcapsule/sandbox:18.11.6-1.42
+host$ docker run ...
+```
+
+### Without Docker
+
+If you choose not to use `Docker`, or if you are using a Linux distribution incompatible with the `Debian` based sandbox, then you need to [install DPDK from source](https://doc.dpdk.org/guides/linux_gsg/build_dpdk.html) yourself.
+
+In addition to the [required tools and libraries](https://doc.dpdk.org/guides/linux_gsg/sys_reqs.html#compilation-of-the-dpdk), you also need the packet captures (pcap) library to run some `Capsule` example applications. Install `libpcap-dev` for `Debian`/`Ubuntu` or `libpcap-devel` for `RHEL`/`CentOS`/`Fedora`.
+
+Once you have installed all the necessary tools and libraries on your system, you should use our [script](scripts/dpdk.sh) to install the version of DPDK that `Capsule` uses.
+
+You can also use another [script](scripts/rustup.sh) to install the latest stable `Rust` if you don't have that setup already. Since kernel 4.0, running `DPDK` applications requires `root` privileges. You must install the `Rust` toolchain for the `root` user as well if you want to run your project with `cargo run`.
+
+We've also provided another VirtualBox VM for development, sans `Docker`, if you don't want to manually install `DPDK`, `Rust`, and the necessary tools and libraries.
+
+```
+host$ vagrant up vm
+host$ vagrant ssh
+```
+
+### Kernel NIC Interface
+
+If your application uses [KNI](https://doc.dpdk.org/guides/prog_guide/kernel_nic_interface.html), you will need the kernel module `rte_kni`. Kernel modules are version specific. We may provide precompiled modules for different kernel versions and Linux distributions in the future. But for now, you will have to compile it yourself by installing the kernel headers or sources required to build kernel modules on your system, and then build `DPDK` from source. Follow the directions above.
+
+Once compiled, you can load it using command,
+
+```
+host$ sudo insmod /lib/modules/`uname -r`/extra/dpdk/rte_kni.ko
+```
+
+## Packaging for Release
+
+When packaging your application for release, the package must include the shared `DPDK` libraries and have as dependencies `libnuma` and `libpcap` for your Linux distribution.
+
+If you want to containerize your release, you can use [`getcapsule/dpdk:18.11.6`](https://hub.docker.com/repository/docker/getcapsule/dpdk) as the base image which includes `libnuma`, `libpcap` and `DPDK`. For other packaging methods, you can find the `DPDK` libraries in `/usr/local/lib/x86_64-linux-gnu`.
+
+## Contributing
+
+Thanks for your help improving the project! We have a [contributing guide](https://github.com/capsule-rs/capsule/blob/master/CONTRIBUTING.md) to help you get involved with the `Capsule` project.
+
+## Code of Conduct
```

Comment:
@drunkirishcoder one other note, can you add https://github.com/capsule-rs/capsule/pull/69/files#diff-04c6e90faac2675aa89e2176d2eec7d8R80 aka `Contact` to here as well?